<ui version="4.0" >
 <class>InternalValidator</class>
 <widget class="QWidget" name="InternalValidator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>79</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Accessibility validation level:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KComboBox" name="accessibilityLevel" >
       <item>
        <property name="text" >
         <string>0 (No check)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QCheckBox" name="runAfterLoading" >
       <property name="whatsThis" >
        <string>If this option is enabled, the internal HTML validator is executed after a webpage has been loaded.</string>
       </property>
       <property name="text" >
        <string>Run after loading</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
